firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/schema.ts

```diff
@@ -149,3 +150,31 @@ export const RawChangelogSchema: any = {
     documentIdField,
   ],
 };
+
+// Helper function for Partitioned Changelogs
+export const getRawChangelogPartitioned = (
+  partitioningField: string,
+  partitioningFieldType: string,
+  isView: boolean
+) => {
+  const isSchemaFieldExist =
+    RawChangelogSchema.fields.find(
+      (field) => field.name === partitioningField
+    ) ||
+    RawChangelogViewSchema.fields.find(
+      (field) => field.name === partitioningField
+    );
+  const newPartitionField = {
+    name: partitioningField,
+    mode: "NULLABLE",
+    type: partitioningFieldType,
+    description: "The document partition field selected by user",
+  };
+  let schema = isView
```

Comment:
const?

Reply:
I am using the `push` method below so I think need to use `let` in this place